app/views/items/new.html.erb

```diff
@@ -0,0 +1,15 @@
+<p><%= link_to 'Merchants Index', '/merchants' %></p>
+<h2>Create New Item</h2>
+<%= form_tag "/merchants/#{@merchant.id}/items" %>
```

Comment:
this is what i was referring to when I was commenting on the items create you have in the items controller. Since you have this going to /merchants/:merchant_id/items, you have access to that merchants id once you submit this form.